app/controllers/products_controller.rb

```diff
@@ -4,7 +4,7 @@ class ProductsController < ApplicationController
   # GET /products
   # GET /products.json
   def index
-    @products = Product.all
+    @products = Product.all.order(:title)
```

Comment:
```suggestion
    @products = Product.order(:title)
```
Simplified but same query.